src/components/article/ArticleProvider.js

```diff
@@ -35,7 +33,7 @@ const getArticleById = (id) => {
         .then(res => res.json())
```

Comment:
I don't think it will hurt anything but the question mark is just hanging out at the end lol